backend/app/controllers/spree/admin/addresses_controller.rb

```diff
@@ -0,0 +1,65 @@
+module Spree
+  module Admin
+    class AddressesController < Spree::Admin::BaseController
+      before_action :initial_user
+
+      def show
```

Comment:
Why do we need a `#show` action?

Reply:
I separated in two actions: one is the get method `show`(when user clicks to tab Adresses) and another method post `update`(when user clicks button **update**)

Reply:
Oh, I see. Understood.